sycl/test/basic_tests/accessor/accessor_size_type.cpp

```diff
@@ -0,0 +1,15 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -fsyntax-only
```

There is nothing to run in this test

Reply:
@AlexeySachkov, right, thanks, done.